pkg/app.go

```diff
@@ -16,6 +17,7 @@ import (
 type App struct {
 	Router *mux.Router
 	DB     *sql.DB
+	HMACSecret string
```

Comment:
use correct type `byte[]`

Reply:
Have typecast in app.go now instead for easy handling